### PR TITLE
feat(language): add XML and Robot language

### PR DESCRIPTION
**Pull request by fearless-spider**

### CODE REVIEW

File: src/language/detect_language.py [hunk 1]
```diff
@@ -59,6 +60,7 @@
     'Swift': ['swift'],
     'TypeScript': ['ts', 'tsx'],
     'Vue': ['vue'],
+    'XML': ['xml'],
```
Comment (peti2001) [inline src/language/detect_language.py:63]: WE don't want to include XML files. Here is why: https://github.com/codersrank-org/repo_info_extractor/pull/130

File: src/language/detect_language.py [hunk 2]
```diff
@@ -47,6 +47,7 @@
     'Protocol Buffer': ['proto'],
     'Python': ['py'],
     'R': ['r'],
+    'Robot': ['robot'],
```
Comment (peti2001) [inline src/language/detect_language.py:50]: Please update the test cases too.

File: src/language/detect_language.py (hunk 1 above)
Comment (fearless-spider) [reply src/language/detect_language.py:63]: I see, ok :)

File: src/language/detect_language.py (hunk 2 above)
Comment (fearless-spider) [reply src/language/detect_language.py:50]: ok, I will update test cases

**Comment by fearless-spider** [comment]
I added test for c++ and robot

**Comment by peti2001** [comment]
Thanks for your contribution

**Comment by peti2001** [comment]
Hi @fearless-spider,

We are planning to rewrite the repo_info_extractor. I collected the plans here #160, feel free to share your thoughts.

**Pull request merged**